src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -73,9 +73,9 @@ public partial interface IImmutableStack<T> : System.Collections.Generic.IEnumer
     public static partial class ImmutableArray
     {
         public static int BinarySearch<T>(this System.Collections.Immutable.ImmutableArray<T> array, int index, int length, T value) { throw null; }
-        public static int BinarySearch<T>(this System.Collections.Immutable.ImmutableArray<T> array, int index, int length, T value, System.Collections.Generic.IComparer<T> comparer) { throw null; }
+        public static int BinarySearch<T>(this System.Collections.Immutable.ImmutableArray<T> array, int index, int length, T value, System.Collections.Generic.IComparer<T>? comparer) { throw null; }
```

Comment:
Can't comment on it, but `IImmutableSet<T>`'s `TryGetValue`... should that have [MaybeNullWhen(false)] on the out?